app/config.py

```diff
@@ -37,6 +37,13 @@ def sl_getenv(env_var: str, default_factory: Callable = None):
     return literal_eval(value)
 
 
+def env(var: str, default: Optional[str] = None) -> Optional[str]:
```

Comment:
hey this function seems to be equivalent to `os.environ.get(var, default)` which is pretty short, should we use it instead?

Reply:
Oh, i hadn't seen that this usage was possible. Removed it.